README.md

```diff
@@ -114,7 +114,7 @@ The `with` and `update` functions are useful for applying functions to values. T
 
 ``` swift
 class MyViewController: UIViewController {
-  let label = with(UILabel()) {
+  let label = update(UILabel()) {
```

Comment:
Thanks! This should actually be `updateObject` to avoid some Swift overload resolution issues.
```suggestion
  let label = updateObject(UILabel()) {
```